src/applications/vaos/new-appointment/redux/actions.js

```diff
@@ -1018,32 +1022,37 @@ export function submitAppointmentOrRequest(history) {
 
       try {
         let requestData;
-        if (isCommunityCare) {
+        if (featureVAOSServiceRequests && isCommunityCare) {
+          requestBody = transformFormToVAOSCCRequest(getState());
+          requestData = await createAppointment({ appointment: requestBody });
+        } else if (isCommunityCare) {
           requestBody = transformFormToCCRequest(getState());
           requestData = await submitRequest('cc', requestBody);
         } else {
           requestBody = transformFormToVARequest(getState());
           requestData = await submitRequest('va', requestBody);
         }
 
-        try {
-          const requestMessage = data.reasonAdditionalInfo;
-          if (requestMessage) {
-            await sendRequestMessage(requestData.id, requestMessage);
+        if (!featureVAOSServiceRequests) {
```

Comment:
We don't need to set request messages or preferences if we're using the VAOS service, as far as I know.